webpack.wordpress.config.js

```diff
@@ -18,16 +22,32 @@ module.exports = {
 		],
 	},
 	plugins: [
-		new LodashModuleReplacementPlugin( {
-			paths: true,
-			shorthands: true,
-		} ),
 		new webpack.DefinePlugin( {
-			'process.env.NODE_ENV': JSON.stringify( 'development' ),
+			'process.env.NODE_ENV': JSON.stringify( env ),
 		} ),
 	],
 	resolve: {
 		extensions: [ '.js', '.jsx' ],
 		modules: [ path.resolve( __dirname ), path.resolve( __dirname, 'node_modules' ) ],
 	},
 };
+
+switch ( env ) {
+	case 'development':
+		config.devtool = 'source-map';
+		config.devServer = {
```

Comment:
Do we use the development server in this case?

Reply:
I haven't actually tried it, but it should be helpful to serve local css while developing a wordpress plugin.